pkg/exporter/sumologicexporter/config.go

```diff
@@ -106,6 +109,16 @@ func CreateDefaultHTTPClientSettings() confighttp.HTTPClientSettings {
 
 func (cfg *Config) Validate() error {
 
+	switch cfg.HTTPClientSettings.Compression {
```

Comment:
It should use `CompressEncoding` value, so we respect actual customer configuration

Reply:
This is the new validation for the `Compress` field I'll keep the validation for the deprecated field `CompressEncoding` as well.

Reply:
ok I'll into this. At present if `CompressEncoding` is set it takes precedence over `Compression` if both are set.

Reply:
I've added a condition for this - https://github.com/SumoLogic/sumologic-otel-collector/blob/zstd-support/pkg/exporter/sumologicexporter/config.go#L131C84-L131C107